Library/Homebrew/utils.rb

```diff
@@ -431,6 +431,7 @@ def which_editor
 end
 
 def exec_editor(*args)
+  puts "Editing #{args.join "\n"}"
```

Comment:
Aesthetically, the changing indentation and that only the first line is prefixed with “Editing” for the atypical case is bothering me more, even if the common case becomes nicer and shorter.

But to be honest, I'll probably be slightly annoyed by this output anyway, no matter how it is styled, because I basically never want to know where the formula I'm editing is and when I do, my editor can tell me quickly and efficiently. But I'll certainly manage as I guess this is a minority opinion.